src/frontend/helperFunctions/removeFromCart.js

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+const removeFromCart = async (id, setCartItems, token) => {
+  try {
+    const response = await axios.delete(`/api/user/cart/${id}`, {
```

Comment:
you store all the api's in a separate file and import it here that will be easy that you dont't make spelling mistake when entering the api path